modules/lang-painless/src/main/java/org/elasticsearch/painless/ir/DeclarationBlockNode.java

```diff
@@ -21,12 +21,13 @@
 
 import org.elasticsearch.painless.ClassWriter;
 import org.elasticsearch.painless.MethodWriter;
+import org.elasticsearch.painless.phase.IRTreeVisitor;
 import org.elasticsearch.painless.symbol.WriteScope;
 
 import java.util.ArrayList;
 import java.util.List;
 
-public final class DeclarationBlockNode extends StatementNode {
+public class DeclarationBlockNode extends StatementNode {
```

Comment:
why is this not final anymore?

Reply:
For consistency and extensibility. None of the nodes are final as they all have the potential to be used as super classes for new nodes. There are couple in the PR that I found going through them that had been overlooked.